Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+/* Only keep the 10 most recent builds. */
+properties([[$class: 'BuildDiscarderProperty',
+                strategy: [$class: 'LogRotator', numToKeepStr: '10']]])
```

Comment:
We can now use `symbol` based version, slightly more expressive:

```groovy
properties( [ buildDiscarder( logRotator(numToKeepStr: '10') ) ])
```

Reply:
Will refactor when I move it to the library